### PR TITLE
[WIP] OpenComputers integration

### PR DESCRIPTION
**Pull request by Xandaros**
Like for [Deep Resonance](https://github.com/McJty/DeepResonance/pull/96), I started working on OC integration for RFTools.

Planned drivers:
- [X] Infuser
- [X] Dialing device
- [x] Matter receiver
- [x] Matter transmitter
- [x] Coal generator
- [ ] Crafters
- [ ] Builder (maybe)
- [x] Powercells
- [ ] Power relay
- [x] RF Monitor
- [x] Liquid Monitor
- [ ] Shield projectors
- [ ] Environmental controller
- [ ] Matter beamer
- [ ] Spawner
- [ ] Block protector
- [ ] Item filter
- [ ] Security manager (maybe)
- [x] Pearl injector
- [x] Endergenic Generator
- [ ] Storage scanner
- [ ] Elevator
- [ ] Booster

So, basically, all of RFTools :P
There are a few things missing, but that's most of the full-sized blocks.

### CODE REVIEW

**Comment by awilliamson** [comment]
:+1:

**Comment by Xandaros** [comment]
The liquid monitor driver currently only reads the first tank. Might want to allow access to all tanks.

**Comment by SuperSimon81** [comment]
I would love this! Im enjoying playing around with the endergonic generator on a public server. Every once in a while i will have to recalibrate the timings on the sequencers though. I have been looking for a block that can read the endergonic generator in the same fashion as the one probe so as to auto-calibrate when needed. Preferably using OpenComputers. Im really looking forward to this project!
